media/com_content/js/admin-articles-modal.js

```diff
@@ -1,55 +1,63 @@
+* OTHERWISE YOUR CHANGES WILL BE REPLACED ON THE NEXT BUILD.
+**/
+
 /**
  * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
  * @license    GNU General Public License version 2 or later; see LICENSE.txt
  */
-(function() {
-	"use strict";
-	/**
-	 * Javascript to insert the link
-	 * View element calls jSelectArticle when an article is clicked
-	 * jSelectArticle creates the link tag, sends it to the editor,
-	 * and closes the select frame.
-	 **/
-	window.jSelectArticle = function (id, title, catid, object, link, lang) {
-		var hreflang = '', editor, tag;
-
-		if (!Joomla.getOptions('xtd-articles')) {
-			// Something went wrong!
-			window.parent.jModalClose();
-			return false;
-		}
-
-		editor = Joomla.getOptions('xtd-articles').editor;
-
-		if (lang !== '')
-		{
-			hreflang = ' hreflang="' + lang + '"';
-		}
-
-		tag = '<a' + hreflang + ' href="' + link + '">' + title + '</a>';
-
-		window.parent.Joomla.editors.instances[editor].replaceSelection(tag);
-		window.parent.jModalClose();
-	};
-
-	document.addEventListener('DOMContentLoaded', function(){
-		// Get the elements
-		var elements = document.querySelectorAll('.select-link');
-
-		for(var i = 0, l = elements.length; l>i; i++) {
-			// Listen for click event
-			elements[i].addEventListener('click', function (event) {
-				event.preventDefault();
-				var functionName = event.target.getAttribute('data-function');
-
-				if (functionName === 'jSelectArticle') {
-					// Used in xtd_contacts
-					window[functionName](event.target.getAttribute('data-id'), event.target.getAttribute('data-title'), event.target.getAttribute('data-cat-id'), null, event.target.getAttribute('data-uri'), event.target.getAttribute('data-language'));
-				} else {
-					// Used in com_menus
-					window.parent[functionName](event.target.getAttribute('data-id'), event.target.getAttribute('data-title'), event.target.getAttribute('data-cat-id'), null, event.target.getAttribute('data-uri'), event.target.getAttribute('data-language'));
-				}
-			})
-		}
-	});
+(function () {
+  'use strict';
+
+  /**
+    * Javascript to insert the link
+    * View element calls jSelectArticle when an article is clicked
+    * jSelectArticle creates the link tag, sends it to the editor,
+    * and closes the select frame.
+    * */
+
+  window.jSelectArticle = function (id, title, catid, object, link, lang) {
+    var hreflang = '';
+    if (!Joomla.getOptions('xtd-articles')) {
+      // Something went wrong!
+      window.parent.jModalClose();
+      return false;
+    }
+
+    var _Joomla$getOptions = Joomla.getOptions('xtd-articles'),
+        editor = _Joomla$getOptions.editor;
+
+    if (lang !== '') {
+      hreflang = 'hreflang="' + lang + '"';
+    }
+
+    var tag = '<a ' + hreflang + ' href="' + link + '">' + title + '</a>';
+    window.parent.Joomla.editors.instances[editor].replaceSelection(tag);
+    window.parent.jModalClose();
```

Comment:
Can you remove this `window.parent.jModalClose()` and instead add `// @TODO Close the modal`

Reply:
@dgt41 should I change in the es6 file or that you have mentioned only?
Because changes replace in next build.

Reply:
Es6